website/docs/r/oss_bucket.html.markdown

```diff
@@ -167,6 +180,8 @@ The following arguments are supported:
 * `storage_class` - (Optional, ForceNew) The [storage class](https://www.alibabacloud.com/help/doc-detail/51374.htm) to apply. Can be "Standard", "IA" and "Archive". Defaults to "Standard".
 * `server_side_encryption_rule` - (Optional, Available in 1.45.0+) A configuration of server-side encryption (documented below).
 * `tags` - (Optional, Available in 1.45.0+) A mapping of tags to assign to the bucket. The items are no more than 10 for a bucket.
+* `versioning` - (Optional, Available in 1.45.0+ and ap-south-1 region) A state of versioning (documented below).
```

Comment:
There is no need to add a tag `ap-south-1 region`. If there are some limitations, please add notes to show them.

Reply:
done